jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -376,6 +397,23 @@ public StackTraceElement toStackTraceElement() {
 					lineNumber);
 		}
 
+		/*[IF Java10]*/
+		@Override
+		public MethodType getMethodType() {
+			if (null == declaringClass) {
+				/*[MSG "K0639","Stack walker not configured with RETAIN_CLASS_REFERENCE"]*/
+				throw new UnsupportedOperationException(com.ibm.oti.util.Msg.getString("K0639")); //$NON-NLS-1$
+			} else {
```

Comment:
The `else` clause isn't required here as the `if` unconditionally throws.

Reply:
True.  Fixed.